ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -144,7 +147,7 @@ export const AnacondaWizard = ({ dispatch, storageData, localizationData, onAddE
     const flattenedStepsIds = getFlattenedStepsIds(stepsOrder);
 
     const { path } = usePageLocation();
-    const currentStepId = path[0] || "installation-language";
+    const currentStepId = !isBootIso ? path[0] || "installation-language" : path[1] || "storage-devices";
```

Comment:
path contains the currnt URL path. path[1] is wrong here.  You still need path[0].